docs/content/doc/usage/https-support.md

```diff
@@ -32,7 +32,7 @@ HTTP_PORT = 3000
 CERT_FILE = cert.pem
 KEY_FILE  = key.pem
 ```
-
+Note that if your certificate is signed by a third party certificate authority (i.e. not self-signed), then cert.pem should contain the certificate chain. The server certificate must be the first entry in cert.pem, followed by the intermediaries in order. The root certificate does not have to be included because the connecting client must already have it in order to estalbish the trust relationship.
```

Comment:
lgtm.

```suggestion
Note that if your certificate is signed by a third party certificate authority (i.e. not self-signed), then cert.pem should contain the certificate chain. The server certificate must be the first entry in cert.pem, followed by the intermediaries in order (if any). The root certificate does not have to be included because the connecting client must already have it in order to estalbish the trust relationship.
```

I think there are some cases when server certificate is signed directly by root CA, so I took an addition there (`if any`).